packages/zapp/console/src/components/Entities/EntityInputs.tsx

```diff
@@ -0,0 +1,226 @@
+  header: {
+    marginBottom: theme.spacing(1),
+  },
+  divider: {
+    borderBottom: `1px solid ${theme.palette.divider}`,
+    marginBottom: theme.spacing(1),
+  },
+  rowContainer: {
+    display: 'flex',
+    marginTop: theme.spacing(3),
+  },
+  firstColumnContainer: {
+    width: '60%',
+    marginRight: theme.spacing(3),
+  },
+  secondColumnContainer: {
+    width: '40%',
+  },
+  configs: {
+    listStyleType: 'none',
+    paddingInlineStart: 0,
+  },
+  config: {
+    display: 'flex',
+  },
+  configName: {
+    color: theme.palette.grey[400],
+    fontSize: '14px',
+    marginRight: theme.spacing(2),
+    minWidth: '95px',
+  },
+  configValue: {
+    color: '#333',
+    fontSize: '14px',
+  },
+  headCell: {
+    fontSize: '14px',
+    color: theme.palette.grey[400],
+  },
+  noInputs: {
+    color: theme.palette.grey[400],
+  },
+}));
+
+interface Input {
+  name: string;
+  type?: string;
+  required?: boolean;
+  defaultValue?: string;
+}
+
+/** Fetches and renders the expected & fixed inputs for a given Entity (LaunchPlan) ID */
+export const EntityInputs: React.FC<{
+  id: ResourceIdentifier;
+}> = ({ id }) => {
+  const styles = useStyles();
+
+  const launchPlanState = useLaunchPlans(
+    { project: id.project, domain: id.domain },
+    {
+      limit: 1,
+      filter: [
+        {
+          key: 'launch_plan.name',
+          operation: FilterOperationName.EQ,
+          value: id.name,
+        },
+      ],
+    },
+  );
+
+  const closure = launchPlanState?.value?.length
+    ? launchPlanState.value[0].closure
+    : ({} as LaunchPlanClosure);
+
+  const spec = launchPlanState?.value?.length
+    ? launchPlanState.value[0].spec
+    : ({} as LaunchPlanSpec);
+
+  const expectedInputs = React.useMemo<Input[]>(() => {
+    const results = [] as Input[];
+    Object.keys(closure?.expectedInputs?.parameters || {}).forEach((name) => {
+      const parameter = closure?.expectedInputs.parameters[name];
+      if (parameter?.var?.type) {
+        const typeDefinition = getInputDefintionForLiteralType(parameter?.var?.type);
+        results.push({
+          name,
+          type: formatType(typeDefinition),
+          required: !!parameter?.required,
+          defaultValue: parameter?.default?.value,
+        });
+      }
+    });
+    return results;
+  }, [closure]);
+
+  const fixedInputs = React.useMemo<Input[]>(() => {
+    const inputsMap = transformLiterals(spec?.fixedInputs?.literals || {});
+    return Object.keys(inputsMap).map((name) => ({ name, defaultValue: inputsMap[name] }));
+  }, [spec]);
+
+  const configs = React.useMemo(
+    () => [
+      { name: t('configType'), value: 'single (csv)' },
+      {
+        name: t('configUrl'),
+        value:
+          'https://raw.githubusercontent.com/jbrownlee/Datasets/master/pima-indians-diabetes.data.csv',
+      },
+      { name: t('configSeed'), value: '7' },
+      { name: t('configTestSplitRatio'), value: '0.33' },
```

Comment:
nit:
@jsonporter I'm a little bit worried, that this data is not coming from anywhere and hardcoded in flyteconcole. Even if it's static we should ideally receive it from somewhere. Otherwise even if one value changed for all launch plans, our static won't reflect it.

Jason - can you create a ticket to fix it later?

Reply:
As far as I remember, the "Configutration" section was planned to be hidden, per latest sync with Jason. It still currently present in UI. can you please check with @jsonporter, and either hide it or fix the spacing between configuration and Launch Plan Version section.
<img width="695" alt="Screen Shot 2022-06-26 at 7 23 48 PM" src="https://user-images.githubusercontent.com/55718143/175826353-d470761d-affe-40f5-afe4-794352ca4c2b.png">
<img width="200" alt="Screen Shot 2022-06-26 at 7 24 05 PM" src="https://user-images.githubusercontent.com/55718143/175826355-d2637011-d507-4ae2-b448-9440a7426926.png">

Reply:
I've removed Configuration section.